tasks/eth/006-MCP-L1/postCheck.s.sol

```diff
@@ -0,0 +1,231 @@
+        require(address(bridge.OTHER_BRIDGE()) == Predeploys.L2_STANDARD_BRIDGE);
+        require(address(bridge.otherBridge()) == Predeploys.L2_STANDARD_BRIDGE);
+        require(address(bridge.superchainConfig()) == prox.SuperchainConfig);
+    }
+
+    /// @notice Asserts that the L2OutputOracle is setup correctly
+    function checkL2OutputOracle() internal view {
+        console.log("Running chain assertions on the L2OutputOracle");
+        L2OutputOracle oracle = L2OutputOracle(prox.L2OutputOracle);
+
+        require(oracle.SUBMISSION_INTERVAL() == cfg.l2OutputOracleSubmissionInterval());
+        require(oracle.submissionInterval() == cfg.l2OutputOracleSubmissionInterval());
+        require(oracle.L2_BLOCK_TIME() == cfg.l2BlockTime());
+        require(oracle.l2BlockTime() == cfg.l2BlockTime());
+        require(oracle.PROPOSER() == cfg.l2OutputOracleProposer());
+        require(oracle.proposer() == cfg.l2OutputOracleProposer());
+        require(oracle.CHALLENGER() == cfg.l2OutputOracleChallenger());
+        require(oracle.challenger() == cfg.l2OutputOracleChallenger());
+        require(oracle.FINALIZATION_PERIOD_SECONDS() == cfg.finalizationPeriodSeconds());
+        require(oracle.finalizationPeriodSeconds() == cfg.finalizationPeriodSeconds());
+        require(oracle.startingBlockNumber() == cfg.l2OutputOracleStartingBlockNumber());
+        require(oracle.startingTimestamp() == l2OutputOracleStartingTimestamp);
+    }
+
+    /// @notice Asserts that the OptimismMintableERC20Factory is setup correctly
+    function checkOptimismMintableERC20Factory() internal view {
+        console.log("Running chain assertions on the OptimismMintableERC20Factory");
+        OptimismMintableERC20Factory factory = OptimismMintableERC20Factory(prox.OptimismMintableERC20Factory);
+
+        require(factory.BRIDGE() == prox.L1StandardBridge);
+        require(factory.bridge() == prox.L1StandardBridge);
+    }
+
+    /// @notice Asserts that the L1ERC721Bridge is setup correctly
+    function checkL1ERC721Bridge() internal view {
+        console.log("Running chain assertions on the L1ERC721Bridge");
+        L1ERC721Bridge bridge = L1ERC721Bridge(prox.L1ERC721Bridge);
+
+        require(address(bridge.OTHER_BRIDGE()) == Predeploys.L2_ERC721_BRIDGE);
+        require(address(bridge.otherBridge()) == Predeploys.L2_ERC721_BRIDGE);
+
+        require(address(bridge.MESSENGER()) == prox.L1CrossDomainMessenger);
+        require(address(bridge.messenger()) == prox.L1CrossDomainMessenger);
+        require(address(bridge.superchainConfig()) == prox.SuperchainConfig);
+    }
+
+    /// @notice Asserts the OptimismPortal is setup correctly
+    function checkOptimismPortal() internal view {
+        console.log("Running chain assertions on the OptimismPortal");
+
+        OptimismPortal portal = OptimismPortal(payable(prox.OptimismPortal));
+
+        address guardian = cfg.superchainConfigGuardian();
+        if (guardian.code.length == 0) {
```

Comment:
On mainnet, guardian should always have code

Reply:
And once https://www.notion.so/oplabs/Proposal-Align-Sepolia-roles-config-with-Mainnet-a1fb54672f514a5aaec65a564671c621?pvs=4 is done, Sepolia will also